deeplab/README.md

```diff
@@ -0,0 +1,209 @@
+<br />
+<p align="center">
+   <img src="https://assets-global.website-files.com/61fd4eb76a8d78bc0676b47d/6375dfb52e6aae05f4ceacc2_Screen%20Shot%202022-11-17%20at%202.13.48%20AM.png" width="50%" height="50%"/>
+</p>
+
+<h2><p align="center">The most efficient recipes for training DeepLabV3 on ADE20K</p></h2>
+
+<h3><p align='center'>
+<a href="https://www.mosaicml.com">[Website]</a>
+- <a href="https://docs.mosaicml.com/">[Composer Docs]</a>
+- <a href="https://docs.mosaicml.com/en/stable/method_cards/methods_overview.html">[Methods]</a>
+- <a href="https://www.mosaicml.com/team">[We're Hiring!]</a>
+</p></h3>
+
+<p align="center">
+    <a href="https://join.slack.com/t/mosaicml-community/shared_invite/zt-w0tiddn9-WGTlRpfjcO9J5jyrMub1dg">
+        <img alt="Chat @ Slack" src="https://img.shields.io/badge/slack-chat-2eb67d.svg?logo=slack">
+    </a>
+    <a href="https://github.com/mosaicml/benchmarks/blob/main/LICENSE">
+        <img alt="License" src="https://img.shields.io/badge/License-Apache%202.0-green.svg?logo=slack">
+    </a>
+</p>
+<br />
+
+# Mosaic DeepLabV3
+This folder contains starter code for training [mmsegmentation DeepLabV3 architectures](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/deeplabv3plus) using our most efficient training recipes (see our [benchmark blog](https://www.mosaicml.com/blog/behind-the-scenes) or [recipies blog](https://www.mosaicml.com/blog/mosaic-image-segmentation) for details). These recipes were developed to hit baseline accuracy on [ADE20K](https://groups.csail.mit.edu/vision/datasets/ADE20K/) 5x faster or to maximize ADE20K accuracy over long training durations. Although these recipes were developed for training DeepLabV3 on ADE20k, they could be used to train other segmentation models on other datasets. Give it a try!
```

Comment:
```suggestion
This folder contains starter code for training [mmsegmentation DeepLabV3+ architectures](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/deeplabv3plus) using our most efficient training recipes (see our [benchmark blog post](https://www.mosaicml.com/blog/behind-the-scenes) or [recipies blog post](https://www.mosaicml.com/blog/mosaic-image-segmentation) for details). These recipes were developed to hit baseline mean Intersection-over-Union (mIoU) on [ADE20K](https://groups.csail.mit.edu/vision/datasets/ADE20K/) 5x faster or to maximize ADE20K mIoU over long training durations. Although these recipes were developed for training DeepLabV3+ on ADE20k, they could be used to train other segmentation models on other datasets. Give it a try!
```

Reply:
Is accuracy -> mIoU pedantic? 😅

Reply:
nope but blog -> blog post is! changed anyway =)

Reply:
Hehe, that change was suggested on my PR, so I wanted to do the same :P